.buildkite/pipeline_for_upload_action_test.yml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+steps:
+  - label: ":ok_hand: If this message is shown, the pipeline upload succeeded"
+    command: echo "Acknowledged."
```

Comment:
We could make this a bit more useful by, for example, uploading with the shared pipeline vars (see https://github.com/bloom/DayOne-Android/pull/4883/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R683) and check that a sample value is available

```
[[ $RUBY_VERSION = $(cat .ruby-version) ]] || false
```

Alternatively we could use a dedicated file so that the test is more straightforward

```
[[ $TEST_VALUE_FOR_PIPELINE_UPLOAD_WITH_ENV_FILE = "this string is defined in the file at .buildkite/test_env" ]] || false
```